### ISSUE TITLE
do load testing for transfer of results from firebase to postgres

### ISSUE THREAD

**Issue by Hagellach37** [mapswipe/python-mapswipe-workers#482]
we've seen that transfer of results from firebase to postgres can be tricky in situations when a lot of mapping happens, e.g. see #480 

@laurentS proposed something like this:

* write a simple function in locust that sends a mock result (it's python, so it should be easy), with the attributes we care about being valid. Maybe have a function that picks between 5 groups for each of 3 projects or something (you want to try to maximize clashes, I think) to assign these results to. Content of the results can be the same, as I don't think it matters. Maybe pick from 1000 randomly generated userIds, so that you end up having "new" users in the process
* we know firebase will handle it, so just bombard it with those for a few minutes, with a result uploaded on average every second (I know locust takes care of the randomness, they have settings you can define in a realistic way, like "1000 users doing about 1 req/minute", that sort of thing), and count how many we send
* then I'd just run the worker, and check that all the results end up in the db. I'd make sure to run the workers at least 2-3 times, not sure why, just a gut feeling.

To summarize what would be needed:

* [ ] create X projects in postgres and Firebase
* [ ] set up Y users in Firebase
* [ ] start the `firebase-to-postgres` workers
* [ ] function that creates a mock results for a given a) project b) user and c) group (the group could be picked similar to how it is done by the app)
* [ ] function that sends the mock results to firebase, e.g. using [locust](https://locust.io/)
* [ ] wait some time to let the workers do their job
* [ ] check that all results have be transfered correctly, e.g. by counting the number of results in the database

We had something similar in the past already:
* https://github.com/mapswipe/python-mapswipe-workers/blob/eea011d4b1875736f80ef4d5084aee398f987478/mapswipe_workers/tests/test_02_mapping.py

**Comment by Hagellach37** [comment]
load testing with Locust has been implemented a while ago
